ultra/ultra/adapters/default_reward_adapter.py

```diff
@@ -97,45 +99,48 @@ def adapt(observation: Observation, reward: float) -> float:
     #     ignore_vehicle_behind=True,
     # )
 
-    speed_fraction = max(0, ego_observation.speed / closest_wp.speed_limit)
+    # Environment reward from SMARTS
+    env_reward /= 100
+
+    # Termination reward
+    ego_reached_goal_reward = 1.0 if ego_events.reached_goal else 0.0
+
+    # Termination penalty
+    ego_collision_penalty = -1.0 if len(ego_events.collisions) > 0.0 else 0.0
+    ego_off_road_penalty = -1.0 if ego_events.off_road else 0.0
+    ego_not_moving_penalty = -1.0 if ego_events.not_moving else 0.0
```

Comment:
The `ego_not_moving_penalty` is not in the original reward. If i recall correctly, this event will occur when the ego vehicle stops for > 60 seconds. In your experiments, have you ever seen an ego vehicle that had to stop for more than 60 seconds? (e.g. in a high-density traffic scenario)

Reply:
resolved offline